src/components/drive/Dashboard.js

```diff
@@ -41,13 +41,15 @@ export default function Dashboard() {
               style={{
                 fontSize: "1.3rem",
                 color: "black",
-                width: "max-content",
-                padding: " .2rem 2rem .2rem .2rem",
-                borderRadius: ".5rem",
+                width: "120px",
+                textAlign:"center",
+                padding: " .2rem .2rem .2rem .2rem",
+                borderRadius: "2px",
                 backgroundColor: "#c2d2df5e",
+                marginLeft:"10px"
               }}
             >
-              Folders
+              {"> "}Folders
```

Comment:
do same {"> "} for `Files` and `Links` also

Reply:
Sure👍